debian/control

```diff
@@ -0,0 +1,63 @@
+Source: tsc
+Section: games
+Priority: optional
+Maintainer: Muammar El Khatib <muammar@debian.org>
+Build-Depends: debhelper (>= 9),
+               libcegui-mk2-dev (>= 0.6.2),
+               libboost-filesystem-dev,
+               libboost-thread-dev,
+               libboost1.61-all-dev,
+               libsdl1.2-dev,
+               libsdl-image1.2-dev,
+               libsdl-mixer1.2-dev,
+               libsdl-ttf2.0-dev,
+               pkg-config,
+               ttf-dejavu,
+               po4a,
+               gperf,
+               bison,
+               libxml++2.6-dev,
+               rake,
```

Comment:
As explained below, `rake` is not needed for building anymore, because `mruby` has its own version of it.